src/Core/Checkout/Customer/SalesChannel/ChangeEmailRoute.php

```diff
@@ -81,6 +82,11 @@ public function change(RequestDataBag $requestDataBag, SalesChannelContext $cont
 
         $this->customerRepository->update([$customerData], $context->getContext());
 
+        $requestDataBag->set('password', null);
+
+        $event = new CustomerChangedEmailEvent($context, $customer, $requestDataBag);
+        $this->eventDispatcher->dispatch($event);
```

Comment:
Hi @MelvinAchterhuis 
Would you please adjust the testcases as well, to ensure this event will be fired.